plugins/inputs/statsd/statsd.go

```diff
@@ -856,6 +852,11 @@ func (s *Statsd) Stop() {
 	s.Unlock()
 }
 
+// IsUDP returns true if the protocol is UDP, false otherwise.
+func (s *Statsd) isUDP() bool {
+	return s.Protocol[:3] == "udp"
```

Comment:
Use `strings.HasPrefix(`, I think this will panic if the protocol is less than 3 chars.